source/includes/limitations-filtering.rst

```diff
@@ -0,0 +1,20 @@
+- Filtering is not supported with :ref:`reversible sync
+  <c2c-api-reverse>`.
+- The destination cluster must not contain user data prior to starting.
+- The destination cluster must not contain the
+  ``mongosync_reserved_for_internal_use`` system database prior to
+  starting.
+- You cannot modify a filter that is in use. To create a new filter, see
+  :ref:`changing the filter <c2c-change-filter>`.
+- If you rename a collection or database that is part of a filter,
+  ``mongosync`` does not continue syncing under the new name.
+- If you rename a collection or database so that the new name matches
```

Comment:
This is incorrect. If a rename happens within the filter within the same DB, then it will continue to sync on the renamed entity.

For renaming to a different DB, there are special rules. See `Renaming Namespaces` in https://docs.google.com/document/d/1vDjeXw8x592d3NLKL5RHL6Nouw64heAL-yod_PyqKME/edit for more info